### PR TITLE
created /corpctf and /corpctf/registration, added b4ckd00r

### PR DESCRIPTION
**Pull request by ctrlaltdev**
Fixes for the PR

### CODE REVIEW

**Comment by ctrlaltdev** [comment]
Updated from https://github.com/secops-space/secops.space/pull/7

**Comment by ctrlaltdev** [comment]
The form fails when trying to send for now because of @bbriggs - which has to set up CORS

File: pages/corpctf/registration.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      <div className='registration'>
+        <h1>CORPCTF Registration</h1>
+        { !this.state.sent ?
+          <div id='abouttobepwned'>
+            <p>We're so good at social engineering that we made you believe you're registering for a CTF, while you're actually just giving us your credentials. So please, go ahead.</p>
+            <form id='dataLeak' onSubmit={this.rootthenoobs}>
+              <fieldset><input required name='nick' type='username' placeholder='IRC nick' /></fieldset>
+              <fieldset><input required name='passwd' type='password' onBlur={this.haveibeenpwnedpasswd} placeholder='passwd' /></fieldset>
+              <fieldset><input required name='email' type='email' placeholder='your@email.lol' /></fieldset>
+              <fieldset><input type='submit' value='what can possibily do a button in a form?' /></fieldset>
```
Comment (jomahoney) [inline pages/corpctf/registration.js]: I'm just curious wtf `what can possibily do a button in a form?` means

File: pages/corpctf/registration.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          font-size: 18px;
+          line-height: 24px;
+        }
+        #dataLeak fieldset {
+          border: none;
+          text-align: center;
+        }
+        #dataLeak input {
+          display: inline-block;
+          padding: 8px 16px;
```
Comment (jomahoney) [inline pages/corpctf/registration.js]: The use of `px` over a responsive unit like `rem` scares me in a big way...

Comment (ctrlaltdev) [reply pages/corpctf/registration.js]: oooooh, you use rem, noice - what's your base size?

File: pages/corpctf/registration.js (hunk 1 above)
Comment (ctrlaltdev) [reply pages/corpctf/registration.js]: Register|Submit|Send - you think it's really not obvious?

File: pages/corpctf/registration.js (hunk 2 above)
Comment (jxhn) [reply pages/corpctf/registration.js]: I'll have a look through this CSS but otherwise everything looks great. Once we have CORS sorted this is GTM

**Pull request merged**